src/main/java/com/google/sps/data/backgroundTaskManagerInterface.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+import java.util.Optional;
+import java.util.List;
+
+/** Interface for the backgroundTaskManager class. */
+public interface backgroundTaskManagerInterface { 
+/**
+   * Checks VM status to make sure they are healthy. 
+   */
+    void checkVM();
+/**
+   * Removes attendee if they haven't been polled in 2 minutes.
```

Comment:
Dont be so specific about the 2 minutes. Let that be implementation details. Remove from this comment.